chart/templates/_helpers.tpl

```diff
@@ -0,0 +1,8 @@
+{{- define "hcloud-cloud-controller-manager.name" -}}
+{{- default $.Chart.Name $.Values.nameOverride | trunc 63 | trimSuffix "-" }}
```

Comment:
The [Helm Docs](https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function) have no example for passing 2 arguments to `default` and instead suggest:

```suggestion
{{- $.Values.nameOverride | default $.Chart.Name | trunc 63 | trimSuffix "-" }}
```

Reply:
I mean I don't really care, but both styles are valid in this case, it seems. The version you see currently is what you get when you run `helm create barf`.

Reply:
Okay, i am fine with either version, I was just worried that it might be invalid syntax